cameo/core/solver_based_model.py

```diff
@@ -634,6 +635,19 @@ def essential_genes(self, threshold=1e-6):
         return essential
 
     @property
+    def S(self):
```

Comment:
suggesting to drop this property in favor of a function `create_stoichiometric_array` as that will make cobrapy merge easier